### ISSUE TITLE
Color shift issue during training

### ISSUE THREAD

**Issue by wxm030** [lvsn/ZeroComp#5]
Thanks for open-sourcing the training code. I trained on the Interiorverse dataset (FOV 120) for 78k steps (batch size 32). The outputs look generally too red, and the LAB (L*a*b*) metrics are worse than both released checkpoints (iv_7days and iv_2days). Any idea what might be causing this?

<img width="862" height="795" alt="Image" src="https://github.com/user-attachments/assets/ec51d159-15be-42ba-874c-b5c4545672d4" />

<img width="1011" height="344" alt="Image" src="https://github.com/user-attachments/assets/99b494e3-4aa8-456d-80e1-405fb3e8a0b0" />

------------------------------------Config yaml------------------------------------------------------------
defaults:
  - sdi_default
  - _self_

train_batch_size: 16 # Train batch size per GPU, in total 2 H100 -> 32
resolution: 512
dataloader_num_workers: 4
output_dir: outputs/iv_fov_120_ddim_${now:%Y-%m-%d_%H-%M-%S}
conditioning_maps: [depth, normal, diffuse, shading, mask]

# When resume from checkpoint
resume_from_checkpoint: latest
learning_rate: 8e-5

enable_xformers_memory_efficient_attention: true
allow_tf32: true
set_grads_to_none: true

num_train_epochs: 10000
dataset_name: interior_verse
dataset_dir: s3://interiorverse/fov_120/
------------------------------------Config yaml------------------------------------------------------------

<img width="1731" height="851" alt="Image" src="https://github.com/user-attachments/assets/5bead3f3-26b0-413e-afba-d6bb8b4ae164" />

**Comment by wxm030** [comment]
It looks like I didn’t train long enough; by 132k steps, the colors were already fairly close to the open-sourced weights.

<img width="902" height="700" alt="Image" src="https://github.com/user-attachments/assets/eb94003a-8dc2-43a7-a481-ef5063261a38" />
